docker-compose.yml

```diff
@@ -55,15 +57,13 @@ services:
     volumes:
       - ./:/root/go/src/github.com/percona/pmm
       - ./Makefile.devcontainer:/root/go/src/github.com/percona/pmm/Makefile:ro # change Makefile in devcontainer
-      - pmm-bin:/root/go/src/github.com/percona/pmm/bin
```

Comment:
it will result in issues with access on linux, and you will have linux binaries in mac, so it is better to use `/root/go/bin` instead